### ISSUE TITLE
Deployment

### ISSUE THREAD

**Issue by prodia4** [swsnu/swpp17-team5#66]
**What?**  
Deploy frontend app, API server, recommendation service.

**Why?**  
Need to open the service to public.

**How?**  
Will use Google App Engine.

**Comment by prodia4** [comment]
Was looking at either AWS Elasticbeanstalk + RDS + Elastic File Storage or Google App Engine + Cloudsql + Cloud Storage. But since Google offers a much better documentation, chose to go with GAE.

**Comment by prodia4** [comment]
Succeeded in deploying both frontend app & API server to sob-front.appspot.com and sob-back.appspot.com. But there are multiple issues to be resolved.

For the set-cookie header sent by sob-back.appspot.com to have effect sob-front.appspot.com, I needed to set SESSION_COOKIE_DOMAIN & CSRF_COOKIE_DOMAIN to '.appspot.com'. But according to ![this post](https://stackoverflow.com/questions/8944974/using-appspot-com-as-a-partial-domain-cookie), 'appspot.com' was added to the public suffix list of domains that modern browsers should not allow cookies to be set for.

Will try set-cookie with a custom domain.
